apps/namadillo/src/utils/index.ts

```diff
@@ -5,7 +5,7 @@ import BigNumber from "bignumber.js";
 import * as fns from "date-fns";
 import { getDefaultStore } from "jotai";
 import { DateTime } from "luxon";
-import internalDevnetAssets from "namada-chain-registry/namadainternaldevnet/assetlist.json";
+import internalDevnetAssets from "namada-chain-registry/_testnets/namadainternaldevnet/assetlist.json";
```

Comment:
We should use the mainnet here. I just didn't use it in the past because it didn't exists. Then, can you remove the comment on line 127 as well :)

Reply:
Done! Thanks for catching that!